README.md

```diff
@@ -269,11 +269,36 @@ Grape will also automatically respond to HEAD and OPTIONS for all GET, and just
 If you want to use ActiveRecord within Grape, you will need to make sure that ActiveRecord's connection pool
 is handled correctly.
 
-The easiest way to achieve that is by using ActiveRecord's `ConnectionManagement` middleware in your
-`config.ru` before mounting Grape, e.g.:
+In the old days you could use ActiveRecord's `ConnectionManagement` middleware in your
```

Comment:
Lets make two sections, one for ActiveRecord 4 and another for ActiveRecord 5 and not write "in the old days" and just provide instructions to what to do? 

What happened to the middleware? Is it really being replaced by custom code?

Reply:
Ok. Actually active record does not implement that anymore. It's up to the client now. So, if you want the same functionality you will need to add a middleware. I will do the changes.

Reply:
I found https://github.com/rails/rails/issues/26947 and https://stackoverflow.com/questions/41400202/replacing-activerecordconnectionadaptersconnectionmanagement-in-activerecord. Looks like someone already made a gem for this, lets use that instead of recommending rolling our own middleware?

Reply:
I support that. Will do the changes. Although I didn't test this. I test the other solution, but seems like pretty solid.